action.yml

```diff
@@ -10,6 +10,12 @@ inputs:
     description: A flag to disable the use of sudo whilst installing CML.
     default: true
     required: false
+  override:
+    description:
+      Override cml install with a git url (e.g. 'npm install
+      git://github.com:iterative/cml.git#6a387a23650f20fd5eb81bc0f63a9bfc6f98882e')
+    required: false
```

Comment:
I would simply check if `version` starts with `git://` instead...

Reply:
npm takes a few options for that, I'll try it out and make those changes.

Reply:
@casperdcl going to use this: https://regex101.com/r/GNrVzD/1
based from https://docs.npmjs.com/cli/v7/commands/npm-install

Reply:
I'd suggest https://regex101.com/r/eW96Dq/1 :)